tests/rptest/tests/acls_test.py

```diff
@@ -556,24 +556,11 @@ def test_describe_acls(self, use_tls: bool, use_sasl: bool,
         def should_pass_w_base_user(use_tls: bool, use_sasl: bool,
                                     enable_authz: Optional[bool],
                                     authn_method: Optional[str]):
-            '''
-            These are the config combinations where `rpk acl list` should pass
-            with the base user. All other tests should fail with ClusterAuthzError
-            with the base user. These combinations were pulled from previous
-            iterations of this ducktape test (i.e., AccessControlListTest.test_describe_acls).
-            See https://github.com/redpanda-data/redpanda/blob/38650a7b96707ce4a13d9879d459b08bb0a59bba/tests/rptest/tests/acls_test.py#L217
-            '''
-            if use_tls and not use_sasl and enable_authz == False and authn_method == 'mtls_identity':
+            if enable_authz is False:
                 return True
-            elif use_tls and not use_sasl and enable_authz == None and authn_method == 'mtls_identity':
+            elif authn_method == 'none':
```

Comment:
nit: People are known to pass 'none' as a string into YAML?

Reply:
Some clarification on the 'none' argument here is needed. I removed the 'none' arg from the matrix and all tests pass.

Reply:
It's an option: https://github.com/BenPope/redpanda/blob/96508680ad2ecb9ff0d49790c040d1bf4b6339f0/src/v/config/broker_authn_endpoint.cc#L18-L27